prow/apis/prowjobs/v1/zz_generated.deepcopy.go

```diff
@@ -1,7 +1,7 @@
 // +build !ignore_autogenerated
 
 /*
-Copyright 2018 The Kubernetes Authors.
+Copyright The Kubernetes Authors.
```

Comment:
hummm so looks like we have updated the boilerplate :-)

Reply:
ugh :man_facepalming:

Reply:
fixed by skipping generated files